src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-subsystems-form/nvmeof-subsystems-form.component.ts

```diff
@@ -51,6 +55,24 @@ export class NvmeofSubsystemsFormComponent implements OnInit {
   ngOnInit() {
     this.createForm();
     this.action = this.actionLabels.CREATE;
+    if (this.router.url.includes('subsystems/(modal:edit')) {
+      this.edit = true;
+      this.action = this.actionLabels.EDIT;
+      this.route.params.subscribe((params: { subsystem_nqn: string; max_ns: any }) => {
+        this.subsystemForm.get('nqn').setValue(params.subsystem_nqn);
+        this.subsystemForm.get('max_namespaces').setValue(params.max_ns);
+        this.subsystemForm.get('nqn').disable();
+        this.subsystemForm.get('max_namespaces').disable();
+        this.nvmeofService
+          .getInitiators(params.subsystem_nqn)
+          .subscribe((initiators: NvmeofSubsystemHost[]) => {
+            if (initiators[0].nqn !== '*') {
+              this.subsystemForm.get('initiators').setValue('add-host');
+              initiators.forEach((init: NvmeofSubsystemHost) => this.addHost(init.nqn));
+            }
```

Comment:
Even if it looks like a smart UX move, as the CLI supports specifying both `*` and specific NQNs, we shouldn't rule out deployments where both things are set... And this makes impossible adding specific NQNs if `*` is already set.

Reply:
If both are allowed then lets do the same from UI too, makes sense 👍🏿